packages/dashboard/src/pages/Wallets/components/Transfer/TransferERC20.tsx

```diff
@@ -1,27 +1,34 @@
 import { MaskTextField } from '@masknet/theme'
-import { Box, Button, Stack } from '@material-ui/core'
+import { Box, Button, IconButton, Stack, Typography } from '@material-ui/core'
 import { memo, useCallback, useEffect, useMemo, useState } from 'react'
 import {
     EthereumTokenType,
+    formatWeiToEther,
     FungibleTokenDetailed,
     isGreaterThan,
     isZero,
     pow10,
     TransactionStateType,
+    useChainId,
     useFungibleTokenBalance,
     useGasPrice,
+    useNativeTokenDetailed,
     useTokenTransferCallback,
 } from '@masknet/web3-shared'
 import BigNumber from 'bignumber.js'
-import { TokenAmountPanel } from '@masknet/shared'
+import { TokenAmountPanel, useRemoteControlledDialog } from '@masknet/shared'
+import TuneIcon from '@material-ui/icons/Tune'
 import { SelectTokenDialog } from '../SelectTokenDialog'
 import { useDashboardI18N } from '../../../../locales'
 import { EthereumAddress } from 'wallet.ts'
+import { WalletMessages } from '@masknet/plugin-wallet'
+import { useNativeTokenPrice } from '../../../../../../maskbook/src/plugins/Wallet/hooks/useTokenPrice'
```

Comment:
Don't direct use maskbook method in dashboard

Reply:
`theme.palette.background.default` is `#f9fafa`, very close to  `#F7F9FA`, I will use it instead.

Reply:
Yes